### PR TITLE
Add metadata for geospatial python

### PR DESCRIPTION
**Pull request by rogerkuou**
Fixing #37 

I started an initial version of the metadata for the geospatial python course. The template is really helpful. I used the existing metadata of other courses for inspiration. And also used the course intro text @fnattino wrote for EGU small workshop.

@fnattino @svenvanderburg could you review this?

### CODE REVIEW

File: dc-geospatial-python/lesson-url.md [hunk 1]
```diff
@@ -0,0 +1 @@
+https://carpentries-incubator.github.io/geospatial-python/
```
Comment (svenvanderburg) [inline dc-geospatial-python/lesson-url.md:1]:
```suggestion
https://github.com/carpentries-incubator/geospatial-python
```
This should be pointing to the github page

File: dc-geospatial-python/schedule.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+<div class="row">
```
Comment (svenvanderburg) [inline dc-geospatial-python/schedule.md:1]: I would suggest to put the online schedule here. This is the schedule that ends up in the workshop website. And next one will be online. See point 4 in https://github.com/esciencecenter-digital-skills/workshop-template#required-information-and-customizing-your-workshop-page for more information on different schedules (but you can also just do as I say ;) )

**Review by svenvanderburg — APPROVED** [review]
Looks very good @rogerkuou! Thanks for adding this. I left 2 minor comments, but feel free to merge after having a look.

File: dc-geospatial-python/schedule.md (hunk 2 above)
Comment (rogerkuou) [reply dc-geospatial-python/schedule.md:1]: Thanks @svenvanderburg ! I am using the online schedule now

**Comment by rogerkuou** [comment]
Thanks @fnattino and @svenvanderburg for the review! I applied Sven's comments. Merging it now.

**Pull request merged**